pkg/crc/machine/bundle/repository.go

```diff
@@ -102,10 +105,23 @@ func (bundle *CrcBundleInfo) createSymlinkOrCopyOpenShiftClient(ocBinDir string)
 }
 
 func (repo *Repository) Extract(path string) error {
-	_, err := extract.Uncompress(path, repo.CacheDir, &extract.Options{
+	dir := filepath.Join(repo.CacheDir, "tmp-extract")
+	_ = os.RemoveAll(dir) // clean up before using it
```

Comment:
I'd use `ioutil.TempDir()` + `defer os.RemoveAll()`

Reply:
If I use a tmp dir given by the fs, it is not guaranteed to be on the same disk as the cache directory.
If I use `ioutil.TmpDir(repo.CacheDir, "")` and hit ctrl+c, crc can't delete them as the name is random. 

I kept this :/